docs/download.md

```diff
@@ -52,812 +52,33 @@ do not have CUDA forward compatibility (for example, GeForce), CUDA 11.5 or late
 need to ensure the minimum driver (450.80.02) and CUDA toolkit are installed on each Spark node.
 
 ### Verify signature
-* Download the [RAPIDS Accelerator for Apache Spark 22.08.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.08.0/rapids-4-spark_2.12-22.08.0.jar)
-  and [RAPIDS Accelerator for Apache Spark 22.08.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.08.0/rapids-4-spark_2.12-22.08.0.jar.asc)
+* Download the [RAPIDS Accelerator for Apache Spark 22.10.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.10.0/rapids-4-spark_2.12-22.10.0.jar)
+  and [RAPIDS Accelerator for Apache Spark 22.10.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.10.0/rapids-4-spark_2.12-22.10.0.jar.asc)
 * Download the [PUB_KEY](https://keys.openpgp.org/search?q=sw-spark@nvidia.com).
 * Import the public key: `gpg --import PUB_KEY`
-* Verify the signature: `gpg --verify rapids-4-spark_2.12-22.08.0.jar.asc rapids-4-spark_2.12-22.08.0.jar`
+* Verify the signature: `gpg --verify rapids-4-spark_2.12-22.10.0.jar.asc rapids-4-spark_2.12-22.10.0.jar`
 
 The output if signature verify:
 
 	gpg: Good signature from "NVIDIA Spark (For the signature of spark-rapids release jars) <sw-spark@nvidia.com>"
 
 ### Release Notes
 New functionality and performance improvements for this release include:
-* Rocky Linux 8 support
-* Ability to build Spark RAPIDS jars for Java versions 9+
-* Zstandard Parquet and ORC read support
-* Binary read support from parquet
-* Apache Iceberg 0.13 support  
-* Array function support: array_intersect, array_union, array_except and arrays_overlap
-* Support nth_value, first and last in windowing function
-* Alluxio auto mount for AWS S3 buckets
-* Qualification tool: 
-  * SQL level qualification
-  * Add application details view
+* Dataproc qualification, profiling, bootstrap and diagnostic tool
+* Databricks custom docker container
+* AQE support on Databricks
+* MultiThreaded Shuffle feature
+* HashAggregate on Array support
```

Comment:
The HashAggregate on Array feature was removed from this release, it will be in 22.12. 
```suggestion
* HashAggregate on Array support
```